tokenomics-explainers/003-rework-foundation.md

```diff
@@ -27,7 +27,7 @@ For clarity, **we use 8-ETH bonds in the comparison sections** to get an apples-
 ### Overview
 There is only one source of revenue for the protocol, the fee charged to rETH holders. In the current system, this is 14%, paid to node operators, who are required to bring a bond of both ETH and RPL. The proposed rework changes things by creating 2 categories of rewards to distribute this revenue:
 - **NOs (ETH) - "Node Operator Commission Share"**. This is paid to all node operators (NOs) in a similar manner as existing validators. One difference is that your eligibility to receive from this share _depends only on your ETH, not on your RPL_ (where previously the node operator commission required an initial minimum requirement of both ETH and RPL). This change allows ETH-Only node operators.
-- **NOs (RPL) - "Voter Share"**. This share is collected from all megapool validators and distributed to participants based on their proportional share of vote-eligible RPL staked in megapools. If you stake RPL as a node operator (NO), your RPL is eligible for voting, so you would receive a proportional share from this pot of revenue. Importantly this revenue grows with rETH supply, where growth would allow the same supply of vote eligible RPL to earn more ETH. 
+- **NOs (RPL) - "Voter Share"**. This share is collected from all megapool validators and distributed to participants based on their proportional share of vote-eligible RPL staked in megapools. If you stake RPL as a node operator (NO), your RPL is eligible for voting, so you would receive a proportional share from this pot of revenue. This revenue also grows with rETH supply, where growth would allow the same supply of vote eligible RPL to earn more ETH. 
 
 Importantly, the same participant may be eligible to benefit from both categories. As an example take the node operator in existing tokenomics, represented in Figure 1 on the left pie chart in blue. You can see how an equivalent participant shows up in blue in the new tokenomics on the right pie chart. Their ETH earns from the Node Operator Commission Share, which is initially 5% in the current proposal. Their staked RPL earns from the Voter Share, which is initially 9% in the current proposal.
```

Comment:
When the third category of rewards got moved from this section it made wording awkward with back to back sentences starting with "Importantly..."